src/rgw/rgw_sal_daos.cc

```diff
@@ -30,6 +30,7 @@
 #include "rgw_bucket.h"
 #include "rgw_compression.h"
 #include "rgw_sal.h"
+#include "daos_fs_sys.h"
```

Comment:
This should not be needed, we do not use anything from `daos_fs_sys.h`, and `daos_fs.h` is already included in `rgq_sal_daos.h`.

Reply:
Fixed.